.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
I don't think this gem should define the Ruby version it should use. (The `gemspec` can have a minimal version though, I guess.)

Reply:
~ because it is a standalone app too.

Reply:
No, only because a _gem_ is a library, and it should not impose hard choices to developers using it as a library.